Dockerfile

```diff
@@ -75,6 +78,11 @@ COPY scheduler-jobs/deploy_data_specific_image /deploy_data_specific_image
 # Give ownership of the conda cache folder to jenkins, to enable installing packages by the user from JupyterLab
 RUN chown -R 1000:1000 /opt/conda/pkgs/cache
 
+# Create a new jupyter config directory to allow users to add JupyterLab extensions 
+RUN mkdir /jupyter_config_dir \
```

Comment:
Is this folder a location that will be configured on JupyterHub on birdhouse? How are the extensions set to be added in this location?

Reply:
Yes the change will be made in birdhouse. Instead of mounting the google drive settings to `.jupyter`, it will be mounted to that location. When using docker spawner with Jupyterhub, docker creates the `.jupyter` with the user root which cause permission denied error when spawning the image.